packages/block-library/src/file/save.js

```diff
@@ -32,6 +33,7 @@ export default function save( { attributes } ) {
 		  fileName.toString();
 
 	const hasFilename = ! RichText.isEmpty( fileName );
+	const typographyProps = getTypographyClassesAndStyles( attributes );
```

Comment:
Is there any particular reason for adding these manually. Isn't this something that the hook adds automatically?

Reply:
It is because I needed the typography class names and inline styles to be on both the the wrapper, and the download button.
If I only use the save props on the wrapper, or if I try to use an experimental selector for the typography in block.json, then the button styles override the user settings from the controls in the block editor.